Control/lib/config/configProvider.js

```diff
@@ -42,4 +42,8 @@ const config = require(configFile);
 Log.configure(config);
 log.info(`Read config file "${configFile}"`);
 
+// Add Grafana URL to frame-src CSP
```

Comment:
Why is there a need to add twice the `iframeCsp`? Here and index?

`index.js` is using the configuration provided by this module

Reply:
good catch, I forgot to remove it